Description: Lower precision to avoid failure on armel, armhf and ppc64el
Author: Andreas Tille <tille@debian.org>
Last-Update: Mon, 09 Oct 2023 13:52:54 +0200
Remark: Since autopkgtest runs on unpatched source this patch is not helpful
        and rather implemented in debian/tests/autopkgtest-pkg-r.hook

--- a/tests/testthat/test_GRangePairs.R
+++ b/tests/testthat/test_GRangePairs.R
@@ -97,7 +97,7 @@ test_that("test_GRangePairs", {
   
   ## test the xmin and pars value
   expect_equal(ans$first$xmin, 117)
-  expect_equal(ans$first$pars, 3.976482, tolerance=1e-5)
+  expect_equal(ans$first$pars, 3.976482, tolerance=1e-4)
   expect_equal(ans$second$xmin, 111)
   expect_equal(ans$second$pars, 3.931556, tolerance=1e-5)
 }
